docs/docs/getting-started/using-files.md

```diff
@@ -17,9 +17,32 @@ Let's take a look at what it looks like to use different types of local and remo
 Local files are the most intuitive: you just pass normal paths directly to Toolchest. In the background, the files are 
 transferred to and from the cloud.
 
-`inputs` takes files or directories. If a directory is passed, all files within the directory are used as input.
-
-`output_path` takes a directory. Output files are written in this directory.
+`inputs` takes file and/or directory path(s).
```

Comment:
`and/or` and `(s)` is hard to parse, I'd keep as something simpler like "`inputs` takes paths to files for directories"